src/eterna/AnnotationManager.ts

```diff
@@ -567,6 +663,105 @@ export default class AnnotationManager {
         this.onTriggerPoseUpdate.emit();
     }
 
+    public showAnnotationDialog(
+        edit: boolean,
+        ranges: AnnotationRange[],
+        modal: boolean,
+        gameMode: GameMode,
+        pose: Pose2D,
+        gameLayer: Container,
+        closeCallback: () => void,
+        panelPos?: Point,
+        annotation?: AnnotationData
```

Comment:
There's enough arguments here that it might make sense to extract into an interface - when calling it with this many parameters, it's a bit hard to intuit. VS Code should be able to do this refactoring for you

Reply:
Good call! Will adjust it to be an interface